matrix-sdk/src/main/java/org/matrix/androidsdk/data/store/MXFileStore.java

```diff
@@ -529,6 +522,9 @@ public void run() {
                                     // extract the room states
                                     mRoomReceiptsToLoad.addAll(listFiles(mStoreRoomsMessagesReceiptsFolderFile.list()));
                                     mPreloadTime = System.currentTimeMillis() - fLoadTimeT0;
+                                    if (mMetricsListener != null) {
+                                        mMetricsListener.onStorePreloaded(mPreloadTime);
```

Comment:
So this will be be called on another thread, maybe we should specify it on the Javadoc